### PR TITLE
Rename Stack to Stacks and Fix Bugs

### PR DESCRIPTION
**Pull request by arpitjasa-db**
This PR renames instances of MLOps Stack to MLOps Stacks for consistency. It also fixes various bugs, and bumps the cluster source version to `0.2`.

### CODE REVIEW

File: template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl [hunk 1]
```diff
@@ -34,7 +34,7 @@ dbutils.widgets.text(
 {{else}}
 # Unity Catalog registered model name to use for the trained mode.
 dbutils.widgets.text(
-    "model_name", "dev.{{template `schema_name` .}}.{{template `model_name` .}}", label="Model Name"
+    "model_name", "dev.{{template `schema_name` .}}.{{template `model_name` .}}", label="Full (Three-Level) Model Name"
```
Comment (mingyu89) [inline template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl:37]: The variable description looks good. I'm more referring to the `model_name` variable name. We can also wait and see if there's customer feedback about the name.

File: README.md [hunk 2]
```diff
@@ -78,42 +78,41 @@ See the generated ``README.md`` for next steps!
...
-### I have an existing ML project. Can I productionize it using this stack?
-Yes. Currently, you can instantiate a new project from the stack and copy relevant components
-into your existing project to productionize it. The stack is modularized, so
+### I have an existing ML project. Can I productionize it using MLOps Stacks?
+Yes. Currently, you can instantiate a new project and copy relevant components
+into your existing project to productionize it. MLOps Stacks is modularized, so
 you can e.g. copy just the GitHub Actions workflows under `.github` or ML resource configs
  under ``{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/resources`` 
-and ``{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/bundle.yml`` into your existing project.
+and ``{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml`` into your existing project.
```
Comment (mingyu89) [inline README.md:98]: 👍 Thanks

File: README.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ Please follow [the instruction](https://docs.databricks.com/en/dev-tools/cli/dat
 
 To create a new project, run:
 
-    databricks bundle init https://github.com/databricks/mlops-stack
+    databricks bundle init mlops-stacks
```
Comment (mingyu89) [inline README.md:50]: This is awesome. Shall we cut a release and update the alias pointing to the release version later?

File: stack-customization.md [hunk 4]
```diff
@@ -60,14 +59,13 @@ You may also want to update developer-facing docs under `template/{{.input_root_
 or `template/{{.input_root_dir}}/docs/ml-developer-guide-fs.md`, which will be read by users of your stack.
 
 ### CI/CD workflows
-The default stack currently has the following sub-components for CI/CD:
+MLOps Stacks currently has the following sub-components for CI/CD:
 * CI/CD workflow logic defined under `template/{{.input_root_dir}}/.github/` for testing and deploying ML code and models
-* Automated scripts and docs for setting up CI/CD under `template/{{.input_root_dir}}/.mlops-setup-scripts/`
 * Logic to trigger model deployment through REST API calls to your CD system, when model training completes.
-  This logic is currently captured in ``template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/model_deployment/notebooks/TriggerModelDeploy.py``
+  This logic is currently captured in ``template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/model_deployment/notebooks/ModelDeployment.py``
```
Comment (mingyu89) [inline stack-customization.md:65]: This issue must be from long time ago 🤦 .

File: doc-images/mlops-stacks.png
Comment (mingyu89) [inline doc-images/mlops-stacks.png:1]: It's 9x larger. I don't think it's a big deal.

File: template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/README.md.tmpl [hunk 5]
```diff
@@ -47,7 +47,7 @@ df = spark.table(
 ).drop("fare_amount")
 
 df.write.mode("overwrite").saveAsTable(
-    {{ if (eq .input_include_models_in_unity_catalog `yes`) }}name="hive_metastore.default.taxi_scoring_sample"
+    {{ if (eq .input_include_models_in_unity_catalog `no`) }}name="hive_metastore.default.taxi_scoring_sample"
```
Comment (mingyu89) [inline template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/README.md.tmpl:50]: 👍

**Review by mingyu89 — APPROVED** [review]
Looks good to me. Adding more reviewers to give us more confidence.

File: template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl (hunk 1 above)
Comment (arpitjasa-db) [reply template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/deployment/batch_inference/notebooks/BatchInference.py.tmpl:37]: Yeah but then we'd have to add an `if/else` condition around every time that `model_name` is used on whether or not UC Models is enabled, and I didn't want to risk missing one of those this close to public preview.

File: README.md (hunk 3 above)
Comment (arpitjasa-db) [reply README.md:50]: Yup! We'll be cutting the MLOps Stacks v0.2 release after this PR merges, and deco team has plans to add this semantic versioning to the bundle aliases.

**Review by vladimirk-db — APPROVED** [review]

**Review by vladimirk-db — APPROVED** [review]

**Review by lennartkats-db — APPROVED** [review]

**Review by zhe-db — APPROVED** [review]
🚀🚀🚀

**Pull request merged**